data_dictionary/data_dictionary.cc

```diff
@@ -290,6 +290,19 @@ storage_options::value_type storage_options::from_map(std::string_view type, std
         }
         return options;
     }
+    if (type == "SHARED_FILESYSTEM") {
+        shared_filesystem options;
+        if (auto it = values.find("dir"); it != values.end()) {
```

Comment:
My personal deduplication factor is 3 or 4 :) In early alternator days I also started creating constants for each keyword from DynamoDB specification, but Nadav requested that I should drop the idea and just use plain strings, and I'm very grateful to him for this idea, otherwise we'd have tons of constants that are only used 2, maybe 3 times, so they'd actually make the source code substantially larger.